<launch>

  <node pkg="loam_interface" type="loamInterface" name="loamInterface" output="screen" required="true">
    <!-- <param name="vehicleHeight" value="0.75" />  -->
    <param name="vehicleHeight" value="0" /> 
    <!-- <param name="stateEstimationTopic" type="string" value="/Odometry" /> -->
    <!-- <param name="registeredScanTopic" type="string" value="/cloud_registered" /> -->
    <param name="stateEstimationTopic" type="string" value="/state_estimation_vehicle" />
    <param name="registeredScanTopic" type="string" value="/registered_scan_vehicle" />
    <!-- <param name="registeredScanTopic" type="string" value="/velodyne_points" /> -->
    <param name="flipStateEstimation" type="bool" value="false" />
    <param name="flipRegisteredScan" type="bool" value="false" />
    <param name="sendTF" type="bool" value="true" />
    <param name="reverseTF" type="bool" value="false" />
  </node>

</launch>
